clock/scheduledJobs/jobCleanup.js

```diff
@@ -16,40 +16,110 @@ const Promise = require('bluebird');
 const kue = require('kue');
 const jobSetup = require('../../jobQueue/setup');
 const jobQueue = jobSetup.jobQueue;
+const conf = require('../../config');
 const rangeByStateAsync = Promise.promisify(kue.Job.rangeByState);
 kue.Job.prototype.removeAsync = Promise.promisify(kue.Job.prototype.remove);
 
 /**
  * Execute the call to clean up jobs
  *
+ * @param {Number} window - the number of jobs to delete in each batch
```

Comment:
It's legal to do it but it's better *not* to use "window" as a variable name only because it's a global variable when running javascript in a browser context.
Why not just "batchSize" or something like that?